Source/Scene/Scene.js

```diff
@@ -2829,44 +2905,51 @@ define([
         }
 
         frameState.creditDisplay.endFrame();
-
-        if (scene.debugShowFramesPerSecond) {
-            if (!defined(scene._performanceDisplay)) {
-                var performanceContainer = document.createElement('div');
-                performanceContainer.className = 'cesium-performanceDisplay-defaultContainer';
-                var container = scene._canvas.parentNode;
-                container.appendChild(performanceContainer);
-                var performanceDisplay = new PerformanceDisplay({container: performanceContainer});
-                scene._performanceDisplay = performanceDisplay;
-                scene._performanceContainer = performanceContainer;
-            }
-
-            scene._performanceDisplay.update();
-        } else if (defined(scene._performanceDisplay)) {
-            scene._performanceDisplay = scene._performanceDisplay && scene._performanceDisplay.destroy();
-            scene._performanceContainer.parentNode.removeChild(scene._performanceContainer);
-        }
-
         context.endFrame();
-        RequestScheduler.update();
-        callAfterRenderFunctions(frameState);
-
-        scene._postRender.raiseEvent(scene, time);
     }
 
     /**
      * @private
      */
-    Scene.prototype.render = function(time) {
+    Scene.prototype.render = function (time) {
+        if (!defined(time)) {
+            time = JulianDate.now();
+        }
+
+        this._preRender.raiseEvent(this, time);
+        this._jobScheduler.resetBudgets();
+
+        var cameraChanged = checkForCameraUpdates(this);
+
+        var shouldRender = !this.requestRenderMode || this._renderRequested || cameraChanged || (this.mode === SceneMode.MORPHING);
+
+        var now = JulianDate.clone(time);
```

Comment:
Use a scratch variable instead, and clone into `_lastRenderTime` inside `if (shouldRender)`.